.circleci/config.yml

```diff
@@ -60,7 +58,7 @@ jobs:
       - add_ssh_keys
       - checkout
       - setup_remote_docker
-      - run: yarn install && DEPLOY_ENV=staging yarn publish_assets
+      - run: yarn install && DEPLOY_ENV=staging yarn publish-assets
```

Comment:
If this is run in a pre-deploy hook we don't need to run it via CI

Reply:
@izakp - should i also remove the `yarn install` step?